package.json

```diff
@@ -16,59 +16,52 @@
   ],
   "main": "highcharts/module.js",
   "scripts": {
-    "dev": "nuxt",
-    "build": "nuxt build",
-    "start": "cross-env node server/index.js",
-    "generate": "nuxt generate",
-    "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
+    "dev": "nuxt-ts",
+    "build": "nuxt-ts build",
+    "start": "nuxt-ts start",
+    "generate": "nuxt-ts generate",
+    "lint:js": "eslint --ext .js,.vue --ignore-path .gitignore .",
+    "lint": "npm run lint:js",
     "test": "cross-env NODE_ENV=test npm run test:specs:cov && npm run test:e2e:cov",
-    "test:specs": "cross-env TEST=specs ava --config specs.config.js",
+    "test:unit": "cross-env TEST=unit ava ./test/specs/**/*",
```

Comment:
While this brief version is indeed pretty, it may come back to bite us later on when we want unit vs. e2e tests. I think it's easier to maintain when the script is instructed to use the specific ava config (the config for unit tests) and in that, it's files prop can specify the glob. That way, even if globs get checked in, but a specific feature is being worked on, it's easy to see in PRs if someone changed the glob. (personal preference, more my opinion)

Reply:
so revert to the initial testing scripts?

Reply:
I think so. That way any testing config changes take place in the respective ava config files. I find it's easier to manage the diffs that way. The test:unit and TEST=unit though can stay as-is because that's more common convention.